guides/calculators/promotion-calculators.md

```diff
@@ -0,0 +1,163 @@
+
+The following calculators are available to use with order promotions only:
+
+- [Tiered flat rate](#tiered-flat-rate)
+- [Flat percent (item total)](#flat-percent-item-total)
+
+### Tiered flat rate
+
+The [`Spree::Calculator::TieredFlatRate` calculator][tiered-flat-rate] provides
+a tiered flat rate discount. This allows you to charge a rate-based discount
+that depends on the order total. For example, you could set a base discount of
+$10, then give a greater discount on orders over $100, $200, and $500:
+
+|   | Tier             | Discount ($) |
+|---|------------------|--------------|
+| 0 | Base             | $10          |
+| 1 | Orders over $100 | $15          |
+| 2 | Orders over $200 | $20          |
+| 3 | Orders over $500 | $25          |
+
+This calculator has the following preferences:
+
+- `base_amount`: The base discount for any order where the promotion applies.
+- `tiers`: A hash where the key is the minimum order total for the tier and the
+  value is the tier discount: `{ tier=>discount_amount }`.
+- `currency`: The currency. The default value is your store's default currency
+  setting.
+
+### Flat percent (item total)
+
+The [`Spree::Calculator::FlatPercentItemTotal`][flat-percent-item-total]
+calculator provides a flat, percentage-based discount on an order. In the
+`solidus_backend` interface administrators can use this calculator by choosing
+the whole-order calculator labeled "Flat Percent". It has the following
+preference:
+
+- `flat_percent`: The percentage that the order should be discounted.
+
+[tiered-flat-rate]: https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/tiered_flat_rate.rb
+[flat-percent-item-total]: https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/flat_percent_item_total.rb
+
+## Line item promotions only
+
+The following calculators are available to use with line item promotions only:
+
+- [Distributed amount](#distributed-amount)
+- [Percent on line item](#percent-on-line-item)
+
+### Distributed amount
+
+The [`Spree::Calculator::DistributedAmount` calculator][distributed-amount]
+provides an amount-based discount on all of the applicable line items in an
+order. The discount amount is distributed across all of the line items.
+
+For example, if your promotion offers a $5 discount and a customer orders a $20
+item as well as a $10 item, then the discount is distributed across both items:
+$3.33 (on the $20 item) and $1.67 (on the $10 item).
+
+This calculator has the following preferences:
+
+- `amount`: The discount amount given if the line items should be discounted.
+- `currency`: The currency. The default value is your store's default currency
+  setting.
+
+### Percent on line item
+
+The [`Spree::Calculator::PercentOnLineItem` calculator][percent-on-line-item]
+provide a percentage-based discount for each applicable line item in an order.
+In the `solidus_backend` interface, this calculator is labeled "Percent Per
```

Comment:
Do you think we should change the label in the admin UI?

Reply:
I think that "Percent Per Item" makes sense in the admin UI. I think I just wanted to note this in case someone was looking for "Percent on Line Item" in the admin and weren't finding it.